pkg/kv/dynamodb/store.go

```diff
@@ -94,18 +96,22 @@ func (d *Driver) Open(ctx context.Context, kvParams kvparams.Config) (kv.Store,
 		return nil, fmt.Errorf("%w: %s", kv.ErrSetupFailed, err)
 	}
 
-	return &Store{
+	s := &Store{
 		svc:    svc,
-		params: params,
-	}, nil
+		params: params}
+
+	s.StartPeriodicCheck()
+	return s, nil
 }
 
 // isTableExist will try to describeTable and return bool status, error is returned only in case err != ResourceNotFoundException
 func isTableExist(ctx context.Context, svc *dynamodb.DynamoDB, table string) (bool, error) {
 	_, err := svc.DescribeTableWithContext(ctx, &dynamodb.DescribeTableInput{
 		TableName: aws.String(table),
 	})
+	const operation = "isTableExist"
 	if err != nil {
+		dynamoFailures.WithLabelValues(operation).Inc()
 		if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == dynamodb.ErrCodeResourceNotFoundException {
 			return false, nil
 		}
```

Comment:
Verify that for common setup - non-exists table is not consider as an error we like to report as an error.